sql/core/src/test/resources/sql-tests/inputs/postgreSQL/limit.sql

```diff
@@ -12,25 +12,24 @@ SELECT '' AS five, unique1, unique2, stringu1
 SELECT '' AS two, unique1, unique2, stringu1
 		FROM onek WHERE unique1 > 60 AND unique1 < 63
 		ORDER BY unique1 LIMIT 5;
--- [SPARK-28330] ANSI SQL: Top-level <result offset clause> in <query expression>
--- SELECT '' AS three, unique1, unique2, stringu1
--- 		FROM onek WHERE unique1 > 100
--- 		ORDER BY unique1 LIMIT 3 OFFSET 20;
--- SELECT '' AS zero, unique1, unique2, stringu1
--- 		FROM onek WHERE unique1 < 50
--- 		ORDER BY unique1 DESC LIMIT 8 OFFSET 99;
--- SELECT '' AS eleven, unique1, unique2, stringu1
--- 		FROM onek WHERE unique1 < 50
--- 		ORDER BY unique1 DESC LIMIT 20 OFFSET 39;
+SELECT '' AS three, unique1, unique2, stringu1
+ 		FROM onek WHERE unique1 > 100
+ 		ORDER BY unique1 LIMIT 3 OFFSET 20;
```

Comment:
Could we also have a few test cases with ORDER BY and LIMIT + OFFSET, where the sum of the limit and offset are just below and just above the `conf.topKSortFallbackThreshold`? If we can include EXPLAIN output in the tests, we can show that the top-K sort gets planned as intended.

Reply:
Let's add it into `PlannerSuite`